libc/src/signal/sigaction.h

```diff
@@ -9,7 +9,7 @@
 #ifndef LLVM_LIBC_SRC_SIGNAL_SIGACTION_H
 #define LLVM_LIBC_SRC_SIGNAL_SIGACTION_H
 
-#include <signal.h>
+#include <hdr/types/struct_sigaction.h>
```

Comment:
this should be `#include "hdr/types/struct_sigaction.h"`

In general `<angle brackets>` should only be used for public headers